src/styles/base/_typography.scss

```diff
@@ -19,6 +19,12 @@ html {
     font: var(--typography-html-font, $body-font-size $font-content);
   }
   quotes: "“" "”";
+
+  @each $lang, $quotes in $localizedQuotes {
+    &:lang(#{$lang}) {
+      quotes: $quotes;
```

Comment:
This property only applies to quotes that are either added through a CSS `content` field or the HTML `q` element, right and normal text content would have to be already localized in the text itself, right?

Reply:
Exactly, yes.